modules/system/assets/js/framework.js

```diff
@@ -267,7 +267,7 @@ if (window.jQuery.request !== undefined) {
              * Custom function, redirect the browser to another location
              */
             handleRedirectResponse: function(url) {
-                window.location.href = url
+                window.location.assign(url)
```

Comment:
@bennothommo you're losing IE 11 compatibility with this feature :) 
https://developer.mozilla.org/en-US/docs/Web/API/Location/assign#Browser_compatibility

If you want to test `location.href`, might use it as: 
```
global.window = Object.create(window);
const url = "http://dummy.com";
Object.defineProperty(window, 'location', {
  value: {
    href: url
  }
});
expect(window.location.href).toEqual(url); 
```
Or

```
delete global.window.location;
global.window = Object.create(window);
global.window.location = {
  port: '123',
  protocol: 'http:',
  hostname: 'localhost',
};
```

Reply:
@w20k I thought that said yes for Internet Explorer? :P

Reply:
@bennothommo I think it's edge only not IE11  😄If I'm correct.

Reply:
@w20k do you have a copy of IE11 to test with? I've only got Edge on my Windows laptop :(

Reply:
@bennothommo on Parallels I think have it, not the latest version, but will test this out!

Reply:
@LukeTowers I would like to raise one concern, should we support or `say` that we still support IE 11?
I've tried testing it on my parallels and here is how it looks...
Can we officially drop IE11?

_Would say it not usable at all with my IE11 version_
<img width="1292" alt="Screenshot 2019-09-03 at 21 48 16" src="https://user-images.githubusercontent.com/1053320/64200262-cf4e2f00-ce94-11e9-9124-268f53e687b2.png">
<img width="1304" alt="Screenshot 2019-09-03 at 21 48 40" src="https://user-images.githubusercontent.com/1053320/64200263-cf4e2f00-ce94-11e9-8f03-b7afbf6452d9.png">
<img width="631" alt="Screenshot 2019-09-03 at 21 48 48" src="https://user-images.githubusercontent.com/1053320/64200264-cf4e2f00-ce94-11e9-8ceb-9777ee91e45e.png">

Reply:
@w20k afaik it should be working fine on IE 11, ping me again in a couple days if I haven't gotten around to running it on a real computer.

Reply:
Will do!

Reply:
@LukeTowers ping-ping, IE 11 test run :)

Reply:
I've been able to confirm that `location.assign()` does exist on IE11.

Reply:
Cool @bennothommo, will review your PR one more time, it's good for merging!